src/main/java/org/apache/datasketches/theta/EmptyCompactSketch.java

```diff
@@ -41,12 +41,12 @@ final class EmptyCompactSketch extends CompactSketch {
 
   private EmptyCompactSketch() {}
 
-  static EmptyCompactSketch getInstance() {
+  static synchronized EmptyCompactSketch getInstance() {
```

Comment:
This is an issue with a private static final instance? Is the concern that it can be modified with reflection?

Reply:
Yes it is possible to circumvent using reflection.  There are more complex solutions which prevent this, but, given that we have never claimed that our library is thread-safe, or immune to reflection I think this simple modification at least satisfies the SpotBugs detected issue so that we are not completely ignoring it.